cloudbuild_release.yaml

```diff
@@ -0,0 +1,57 @@
+# Copyright 2019 Google Inc.  All Rights Reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+# This is meant to be used by the Build Trigger set in Cloud Build for auto
+# release. This is what it does:
+# - Builds an image using the last GitHub commit hash as the tag.
+# - Pushes that image to Container Registry.
+# - Runs all integration test from that very image.
+# - Pushes the verified image to the main project (gcp-variant-transforms).
+
+steps:
+  - name: 'gcr.io/cloud-builders/docker'
+    args:
+      - 'build'
+      - '--tag=gcr.io/${PROJECT_ID}/gcp-variant-transforms:${COMMIT_SHA}'
+      - '--file=docker/Dockerfile'
```

Comment:
args: [ 'build', '-t', 'gcr.io/${PROJECT_ID}/gcp-variant-transforms:${COMMIT_SHA}', 'docker/' ]

you can break it out on lines if you want, but the `-t` and dropping `--file` should help make it more concise